week-3/Homework/mandatory/5-journey-planner.js

```diff
@@ -120,9 +138,28 @@ function getLocationName() { }
   Advanced challange: try to use arrow function when invoking an array method.
 */
 function journeyPlanner(locations, transportMode) {
-  // Implement the function body
+
+  let destination = locations.map(function (transportation) {
```

Comment:
You're very close here - you need to remember that `.map()` always returns an array of the same length. So you can't use `.map` to filter the array and to get the location names at the same time. I can talk you through this one if you'd like - I can organise a meeting with the others in the group.